benchmark/run_benchmark.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+REPO=https://github.com/fetchai/agents-aea.git
+BRANCH=feature/benchmarks
+TMP_DIR=$(mktemp -d -t bench-XXXXXXXXXX)
+git clone --branch $BRANCH $REPO $TMP_DIR
+
+CURDIR=`pwd`
+cd $TMP_DIR
+pip install --upgrade aea
+pipenv lock --requirements > requirements.txt
+pip install -r requirements.txt
```

Comment:
? why

Reply:
cause it uses `psutil = "==5.7.0"` that is part of dev requirements
so it's better to install aea and all development requirements